docs/digger/admin/digger-nagios-monitoring.asciidoc

```diff
@@ -0,0 +1,7 @@
+== Digger Nagios Monitoring
+Nagios is provisioned by default by the Digger Ansible playbook. The Nagios instance will perform a number of checks (WHEN???):
```

Comment:
@aidenkeating I think in this case we can say that 'Once Nagios is successfully installed a number of checks triggered by the playbook.